app/models.py

```diff
@@ -30,6 +32,8 @@ class Service(db.Model):
                                nullable=False)
     data = db.Column(JSON)
 
+    status = db.Column(db.String, index=False, unique=False, nullable=False)
```

Comment:
Had a look at some work around enums, seems a little tricky to get the full benefits: see http://techspot.zzzeek.org/2011/01/14/the-enum-recipe/

Going to miss it out for now, but one to come back too.